standalone/ansible/roles/simple-server/templates/.env.j2

```diff
@@ -1,5 +1,6 @@
 DEFAULT_NUMBER_OF_RECORDS=100
 DEFAULT_COUNTRY={{ default_country }}
+MAILERS_FROM={{ mailers_from }}
```

Comment:
Can we just make use of `help@{{domain_name}}`? The more per host variables we add, the harder it gets to keep track of things that need to be updated.